src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        String[] mashine = new String[] {"Truck", "Bulldozer", "Excavator"};
```

Comment:
The variable name `mashine` contains a typo and is not used in the code. If it's not needed, it should be removed to keep the code clean.